### PR TITLE
Fix #2686: Aligned home Activity component Recently played story items in landscape mode.

### LINKED ISSUE #2686: Recently played story item not aligned in landscape mode
**Describe the bug**
The recently played story item is not properly aligned for landscape layout.

**To Reproduce**
1. Go to HomeActivity
2. Change configuration to landscape
3. Observe that there's a little extra space before the recently played story item

**Expected behavior**
There should be no extra space before the story item.

**Screenshots**
Landscape (To be fixed)

Portrait-

**Device**
- Device - Redmi K20 Pro
- API Level -29

### PR DESCRIPTION
**Pull request by Akshat-gour**
## Explanation
This PR Fixes #2686: This includes a change in land/dimens.xml value of home_padding_start is changed, which change the configuration to landscape of Recently played story items.
## After Screen Shot

**Tablet**
image
image

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/values-land/dimens.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
   <dimen name="home_padding_end">72dp</dimen>
-  <dimen name="home_padding_start">72dp</dimen>
+  <dimen name="home_padding_start">65dp</dimen>
```
Comment (FareesHussain) [inline app/src/main/res/values-land/dimens.xml]: According to [Working on UI](https://github.com/oppia/oppia-android/wiki/Working-on-UI#paddingmarginview-dimensions) we recommend using dimensions which are multiples of 4

**Review by FareesHussain — CHANGES_REQUESTED** [review]
@Akshat-gour PTAL

File: app/src/main/res/values-land/dimens.xml (hunk 1 above)
Comment (anandwana001) [reply app/src/main/res/values-land/dimens.xml]: @Akshat-gour Conversation should be resolved only by reviewers or the comment author.

Comment (Akshat-gour) [reply app/src/main/res/values-land/dimens.xml]: Done

Comment (anandwana001) [reply app/src/main/res/values-land/dimens.xml]: as per here as the comment author is @FareesHussain , so he can confirm the conversation and look for what changes suggested and mark the conversation resolved. 

The PR author should not mark the conversation resolved, as you @Akshat-gour are the PR author here.

Comment (Akshat-gour) [reply app/src/main/res/values-land/dimens.xml]: @FareesHussain 
please review it. I made changes as suggested

Comment (FareesHussain) [reply app/src/main/res/values-land/dimens.xml]: You can mark this resolved now.

**Review by FareesHussain — APPROVED** [review]
LGTM

**Comment by Akshat-gour** [comment]
@rt4914 please review the PR

**Review by rt4914 — CHANGES_REQUESTED** [review]
Thanks @Akshat-gour your solution seems correct but I have found this alignment difference in mocks itself so just confirming this with designer about the final UI. Till then you can work on other issues.

@mschanteltc Have a look at the below screenshot. I have following questions related to this:
1. `Recently-Played Stories` is not aligned with `All topics`.
2. `Recently-Played Stories` is not aligned with `Promoted Story Card`.
3. `All Topics` seems to be aligned with `Place Values` topic card.

Is everything in mock correct or does this needs to be changed?
Screenshot 2021-02-15 at 5 43 32 PM


Note: Open the above screenshot in separate screen for better visibility with black background.

Mock Reference: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/0dbdd000-6738-4f6b-9026-0d16737b7baa/specs/

**Comment by mschanteltc** [comment]
I changed points 1 and 2. Point 3 should be kept the same. ([mock](https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/0dbdd000-6738-4f6b-9026-0d16737b7baa/)) In summary, the following should apply:

1. `Recently-Played Stories` is aligned with `All topics`.
2. `Recently-Played Stories` is aligned with `Promoted Story Card`.
3. All Topics is aligned with Place Values topic card.

**Comment by Akshat-gour** [comment]
@rt4914 PTAL

**Comment by rt4914** [comment]
> I changed points 1 and 2. Point 3 should be kept the same. ([mock](https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/0dbdd000-6738-4f6b-9026-0d16737b7baa/)) In summary, the following should apply:
> 
> 1. `Recently-Played Stories` is aligned with `All topics`.
> 2. `Recently-Played Stories` is aligned with `Promoted Story Card`.
> 3. All Topics is aligned with Place Values topic card.

Thanks @mschanteltc 

@Akshat-gour Can you please fix the overall UI so that all three cases work.

**Comment by Akshat-gour** [comment]
@rt4914 
I guess we are talking about Home Activty in Landscape mode
I don't know about **Promoted Story Card**
On home Activity  there are only **Recently Played Story** and **All topics**

**Comment by Akshat-gour** [comment]
@rt4914
I guess we are talking about Home Activty in Landscape mode
I don't know about Promoted Story Card
On home Activity there are only Recently Played Story and All topics

**Comment by FareesHussain** [comment]
@Akshat-gour Promoted Story card is the same as Recently Played Story.
In this https://github.com/oppia/oppia-android/pull/2723#issuecomment-780216116 to be more specific
> 2. Recently-Played Stories is aligned with Promoted Story Card. 

Recently-Player Stories (Title) and Promoted Story Card (Items of the Horizontal Recyclerview).

**Comment by Akshat-gour** [comment]
> > I changed points 1 and 2. Point 3 should be kept the same. ([mock](https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/0dbdd000-6738-4f6b-9026-0d16737b7baa/)) In summary, the following should apply:
> > 
> > 1. `Recently-Played Stories` is aligned with `All topics`.
> > 2. `Recently-Played Stories` is aligned with `Promoted Story Card`.
> > 3. All Topics is aligned with Place Values topic card.
> 
> Thanks @mschanteltc
> 
> @Akshat-gour Can you please fix the overall UI so that all three cases work.

Done

@rt4914 
@FareesHussain 
PTAL 
also updated the screen short

**Review by FareesHussain — APPROVED** [review]
LGTM

**Comment by rt4914** [comment]
@mschanteltc In below images what should be the alignment.
Should the last card in carousel align with the `VIEW-ALL` end. or should the left cards align with the `Stories for you / Recently Played Stories`

**Comment by mschanteltc** [comment]
> @mschanteltc In below images what should be the alignment.
> Should the last card in carousel align with the `VIEW-ALL` end. or should the left cards align with the `Stories for you / Recently Played Stories`
> 
> 

Have them align with the first choice where "last card in carousel aligns with the `VIEW-ALL` end"

**Comment by rt4914** [comment]
@Akshat-gour Please fix the above suggestion too in this PR.

**Comment by Akshat-gour** [comment]
@rt4914 
@FareesHussain 
PTAL
Also updated the screen shots

**Review by FareesHussain — APPROVED** [review]
LGTM, thanks.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@Akshat-gour Thanks for his but the solution works only for mobile. Please make similar changes for `Tablet` devices too.

**Comment by Akshat-gour** [comment]
@rt4914 
@FareesHussain 
i did for tablet and updated the screen shot

**Comment by Akshat-gour** [comment]
@rt4914
@FareesHussain
i did for tablet and updated the screen shot
PTAL

**Review by FareesHussain — APPROVED** [review]
LGTM, thanks

**Review by rt4914 — APPROVED** [review]
Nice work @Akshat-gour

**Pull request merged**